src/Symfony/Component/ExpressionLanguage/Token.php

```diff
@@ -30,11 +30,11 @@ class Token
     const PUNCTUATION_TYPE = 'punctuation';
 
     /**
-     * @param string $type   The type of the token (self::*_TYPE)
-     * @param string $value  The token value
-     * @param int    $cursor The cursor position in the source
+     * @param string           $type   The type of the token (self::*_TYPE)
+     * @param string|int|float $value  The token value
```

Comment:
`|null` as the previous typehint was nullable too

Reply:
`scalar|null`?

Reply:
I added null because scalar seems to not be recognize  and I could not find it in php doc @nicolas-grekas